docs/workforce-management-framework/topics/pro-installing-angular-dependencies.adoc

```diff
@@ -0,0 +1,16 @@
+[id='pro-installing-angular-dependencies-{chapter}']
+= Installing Angular Dependencies
+
+.Prerequisites
+
+. xref:pro-cloning-the-git-repository-{chapter}[Cloning the Git Repository]
```

Comment:
:) I'm sure this modification had you concerned.

They have been pulled up to User Story Level:
[
getting-started-with-raincatcher.adoc
* node and npm installed (tested on node {WFM-RC-NodeVersion})
* git installed
* access to `git@github.com:feedhenry-raincatcher/raincatcher-angularjs.git`
* MongoDb installed
* Redis installed
]